### DISCUSSION TITLE
Feature Request: Links to notes - come back to an important note you've written

### DISCUSSION THREAD

**Discussion by mbigras** [standardnotes/forum#2701] [Suggestions]
The user story is: I'd like to write a note and create a link back to my note.

Then I'd be able to email it to myself or otherwise place it somewhere else such that I'll be able to easily get back to my note either on the web client or on the mobile client.

As it stands now there is no way for me to create a pointer to my note. I just create my note then hope that I'll remember exactly how to get back to it. I can't create a way to click a link and get right back to where I was in the browser, or touch a link and open where I was in the iOS client.

This feature is in line with the standard note's ideas about simplicity and lack of features. If we're able to easily link to our notes then we don't need to implement reminders, or notifications, or trello-like workboards. Instead we can just create notes with links and place the links anywhere and get back to the thing we actually care about: our note.

**Comment by JaspalSuri** [comment]
Hi Max! Thanks for the feedback! I'll pass it along for review. :)
